MarkdownFile.js

```diff
@@ -996,6 +1006,10 @@ MarkdownFile.prototype._localizeNode = function(node, message, locale, translati
             if (node.title) {
                node.title = this._localizeString(node.title, locale, translations);
             }
+            if (node.url && node.localizable) {
+                // don't pseudo-localize URLs
+                node.url = this._localizeString(node.url, locale, translations, true);
```

Comment:
Does it makes sense to pseudo-localize urls using some alternative approach, for example: keep the url as is and append an anchor part to it, e.g. `https://google.com` => `https://google.com#pseudo`.

This will allow to verify that links are indeed being localized while keeping the links in a working state.

Reply:
Pseudo is not used very much... I guess we could do that if someone requests it. For now, I'd like to keep it simple.